test/functional/legacy/003_cindent_spec.lua

```diff
@@ -4718,4 +4718,44 @@ describe('cindent', function()
       JSEND
       ]=])
   end)
+
+  it('start of define / vim-patch 8.0.0148', function()
```

Comment:
a more appropriate name would be "line continuations / vim-patch 8.0.0148". I'll change it, just noting for future reference.